torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1543,6 +1538,21 @@ def sample_inputs_masked_select(op_info, device, dtype, requires_grad):
                    dtypesIfCPU=floating_types_and(torch.bfloat16),
                    dtypesIfCUDA=floating_types_and(torch.half),
                    assert_autodiffed=True),
+    HermitianOpInfo('cholesky',
+                    op=torch.cholesky,
+                    dtypes=floating_and_complex_types(),
+                    test_inplace_grad=False,
+                    check_batched_gradgrad=False,
+                    supports_tensor_out=True,
+                    sample_inputs_func=sample_inputs_linalg_cholesky,
+                    decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],
+                    skips=(
+                        # These tests do not take into account custom op.get_op()
```

Comment:
This is fine although it'd be nice if HermitianOpInfo applied this skip so each HermitianOpInfo didn't need to.

Reply:
I agree, I moved this skip to HermitianOpInfo and applied a similar change for TriangularOpInfo.